Fix issue with config reload when using a log pipeline with a metric stage (#6971)
grafana/agent: docs/sources/flow/reference/components/loki.process.md

| metric.gauge     | [metric.gauge][]     | Defines a `gauge` metric.     | no       |
| metric.histogram | [metric.histogram][] | Defines a `histogram` metric. | no       |

{{< admonition type="note" >}}

> If the configuration file for {{< param "PRODUCT_ROOT_NAME" >}} is reloaded, the metrics will be reset. 

Reviewer: ```suggestion
The metrics will be reset if you reload the {{< param "PRODUCT_ROOT_NAME" >}} configuration file.
```

Switch it around to make it more active.
